### DISCUSSION TITLE
Weapon Familiarity

### DISCUSSION THREAD

**Discussion by dragonknightksr** [YimMenu/HorseMenu#202] [Ideas]
Hello guys, I just realize that we have secret weapon stat called Weapon Familiarity, and it's hidden in online mode but does exists. But there is no option to max out that stat by just a click in any menu even in offline menus, can we try to find it and add that option?

Thanks!

**Comment by Rxann** [comment]
> Hello guys, I just realize that we have secret weapon stat called Weapon Familiarity, and it's hidden in online mode but does exists. But there is no option to max out that stat by just a click in any menu even in offline menus, can we try to find it and add that option?
> 
> 
> 
> Thanks!

Is it hidden because it might not do anything in online? I am not familiar with any game mechanics that would use such a thing like horse bonding for example. There is another issue you can refer to that was a request for a stat editor. It says something about the stats not being stats like in GTA.

**Reply by tuyilmaz** [reply]
It does have an effect online, it increases reload speed and accuracy.

**Reply by Rxann** [reply]
Ah okay, do you know if this can get us detected online?

**Reply by dragonknightksr** [reply]
I don't know for sure, but I think it won't get detected because it have nothing to do with money, some kind of chest spawning, etc... What will cause us get detect.

**Reply by tuyilmaz** [reply]
> Ah okay, do you know if this can get us detected online?

I don't know how it works. If it's like other stats we'll be fine.
